docs/rules/no-istanbul-ignore-file.md

```diff
@@ -0,0 +1,30 @@
+# Disallow disabling coverage for whole files (no-istanbul-ignore-file)
```

Comment:
Should the documentation suggest manipulation of the `include` or `exclude` configuration options?  Maybe in a follow-up?